Frontend/ui-customer/src/Components/QuoteListModel/View.purs

```diff
@@ -220,39 +221,58 @@ findingRidesView state push =
 
 selectRideAndConfirmView :: forall w . QuoteListModelState -> (Action  -> Effect Unit) -> PrestoDOM (Effect Unit) w
 selectRideAndConfirmView state push = 
-  linearLayout
-  [ height WRAP_CONTENT
+  frameLayout
+  [
+    height WRAP_CONTENT
   , width MATCH_PARENT
   , background Color.white900
-  , visibility if ( null state.quoteListModel ) then GONE else VISIBLE
-  ][ linearLayout[
-    height WRAP_CONTENT
-  , weight 1.0
-  , padding (Padding 16 16 0 16)
-  ][textView
-    [ height WRAP_CONTENT
-    , color Color.black900
-    , textSize FontSize.a_16
-    , fontStyle $ FontStyle.medium LanguageStyle
-    , text case getValueToLocalStore AUTO_SELECTING of
-       "CANCELLED_AUTO_ASSIGN" -> "Select a Ride"
-       "false"                 -> "Select a Ride"
-       _                       -> case (getValueToLocalStore LANGUAGE_KEY) of
-                                    _ -> "Confirming selected ride in" <> " : " <> (fromMaybe configDummy ((filter (\item -> item.id == (fromMaybe "" state.selectedQuote)) state.quoteListModel) !! 0)).timer <> "s"
-                                    -- _ -> "state.timer" <> "s " <> (getString AUTO_ACCEPTING_SELECTED_RIDE) TODO :: NEED TO UPDATE LANGUAGE
-    ]]
-   , linearLayout
-    [ height MATCH_PARENT
-    , width WRAP_CONTENT
-    , gravity CENTER
-    , padding (Padding 16 12 16 12)
-    , onClick push $ const CancelAutoAssigning
-    ][ imageView
-      [ height $ V 24
-      , width $ V 24
-      , imageWithFallback "ny_ic_close,https://assets.juspay.in/nammayatri/images/common/ny_ic_close.png"
-      , visibility if getValueToLocalStore AUTO_SELECTING == "false" || getValueToLocalStore AUTO_SELECTING == "CANCELLED_AUTO_ASSIGN" then GONE else VISIBLE 
-     
+  , visibility if ( null state.quoteListModel ) then GONE else VISIBLE 
+  ][
+    PrestoAnim.animationSet 
+    [ translateInXAnim $ translateInXAnimConfig (getAnimationDuration "") (calculateStartPosition "") (not ( null state.quoteListModel))] $ linearLayout
+    [ height if os == "IOS" then (V 60) else MATCH_PARENT
+    , width MATCH_PARENT
+    , background Color.blue700'
+    , alpha 0.2
+    , visibility if getValueToLocalStore AUTO_SELECTING == "false" || getValueToLocalStore AUTO_SELECTING == "CANCELLED_AUTO_ASSIGN" then GONE else VISIBLE 
+    ][]
+  , linearLayout
+    [ height if os == "IOS" then (V 60) else MATCH_PARENT
```

Comment:
Formatting and indentation is not proper here

Reply:
done